docs/installation.rst

```diff
@@ -19,6 +19,28 @@ You can also install it into a new virtual environment:
    $ source ~/.virtualenvs/cernopendata-client/bin/activate
    $ pip install cernopendata-client
 
+If you would like to use a more performing **pycurl** library for downloading data, please add the `pycurl` flavour while installing `cernopendata-client`:
+
+.. code-block:: console
+
+   $ pip install cernopendata-client[pycurl]
+
+Note that you need to have `curl <https://curl.se/>`_ and `openssl <https://www.openssl.org/>`_ set up on your system.
+
+Additionally, if you want to use an even more powerful **XRootD** protocol for downloading data, please add the `xrootd` flavour when installing `cernopendata-client`:
+
+.. code-block:: console
+
+   $ pip install cernopendata-client[xrootd]
+
+Note that you need to have `xrootd <https://xrootd.slac.stanford.edu/>`_ set up on your system.
+
+Finally, note that you can combine both flavours, if you wish to have both capabilities.:
```

Comment:
`capabilities:`  (i.e. no dot before punctuation)

P.S. py37 test still fails?

Reply:
Test is failing because the checksum is not matching in `test_download_files_http_pycurl/requests`. It is working fine on my machine. Looking into it.